pages/how-to-support/become-volunteer.js

```diff
@@ -42,7 +42,7 @@ const BecomeVolunteer = ({
     <Head title={metaTitle} description={metaDescription} />
 
     <Hero
-      imageUrl="/static/images/hero/hero-small-aktiv-werden.jpg"
+      imageUrl="/static/images/hero/hero-small-choir.jpg"
```

Comment:
You didn't remove the actual image from the project is there a reason for that?

Reply:
Done.